pkg/log/logger.go

```diff
@@ -0,0 +1,155 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package log
+
+import (
+	"sync"
+
+	"github.com/trustbloc/edge-core/pkg/internal/logging/metadata"
+)
+
+//nolint:lll
+const (
+	// loggerNotInitializedMsg is used when a logger is not initialized before logging
+	loggerNotInitializedMsg = "Default logger initialized (please call log.Initialize() if you wish to use a custom logger)"
+	loggerModule            = "edge-core/pkg/log"
```

Comment:
I updated this loggerModule constant otherwise everything else is the same as the one in Aries-Framework-Go.